### ISSUE TITLE
Hooks `beforeOnCellMouseOver` and `afterOnCellMouseOver` doesn't work as expected

### ISSUE THREAD

**Issue by wszymanski** [handsontable/handsontable#4296]
### Description

Hooks aren't triggered when hovering cells in described below sequence.

### Steps to reproduce

1. Open http://jsfiddle.net/vxnv04qz.
2. Mouse over a cell.
3. Mouse out the cell.
4. Mouse over the same cell again.

**Result**: Events `beforeOnCellMouseOver` and `afterOnCellMouseOver` aren't triggered (console.log doesn't appear).

**Probable solution**: clear variable inside `afterOnCellMouseOut` event (handsontable/src/3rdparty/walkontable/src/event.js).

```
const mainWOT = that.instance.cloneSource || that.instance;
mainWOT.lastMouseOver = null;
```

### Demo

http://jsfiddle.net/vxnv04qz/

### Your environment
* Handsontable version: 0.32.0
* Browser Name and version: 58.0
* Operating System: 10.12.5

Inform [forum#5496](https://forum.handsontable.com/t/beforeoncellmouseover-callback-issue/5496)

**Comment by AMBudnik** [comment]
actually, we have two `onCellMouseOver` hooks
docs: https://docs.handsontable.com/pro/1.11.0/Hooks.html#event:afterOnCellMouseOver
One at line `304` and second at `318` in the handsontable/src/pluginHooks.js file

**Comment by wszymanski** [comment]
@AMBudnik Thanks. I've also seen it. It will be changed by next update. @swistach has done it within https://github.com/handsontable/handsontable/commit/6e65b1d31b2db83f47d8512bf5564e7deb39a3d3 :)

**Comment by AMBudnik** [comment]
Great! Thanks for a quick update

**Comment by AMBudnik** [comment]
Replicable in v0.35.0/PRO 1.15.0
http://jsfiddle.net/nfsujbg3/

**Comment by adrianszymanski89** [comment]
Inform https://github.com/handsontable/handsontable/discussions/9615#discussioncomment-3062506

**Comment by AMBudnik** [comment]
@d-haber I saw that you left a 👍 for this bug. So I am happy to announce that it was fixed in v12.4.0 released just an hour ago. 

I am closing this issue as solved.

**Comment by d-haber** [comment]
Thank you!  It was just reported in our app about a week ago.  I guess we had perfect timing!  ;-)

Thanks again.
